.github/workflows/github-actions-after-push.yml

```diff
@@ -0,0 +1,21 @@
+name: GitHub Actions After Push Into Develop Branch
+on:
+  push:
+    branches:
+      - develop
+jobs:
+ cypress-run:
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+      - name: Install node
+        uses: actions/setup-node@v2
+        with:
+           node-version: '14'
+      - name: Install dependencies
+        run: npm install
+      - name: Cypress run
+        run: npx cypress run
```

Comment:
Change this run: script from "npx cypress run" into npm "run" from package.json file (after you'll add it in an appropriate file).